.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,70 @@
+name: Bug 🐞
+description: Report a bug report
+labels: [kind/bug 🐞]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a bug report, please search for the behaviour in the existing issues.
+
+        ---
+
+        Thank you for taking the time to file a bug report. To address this bug as fast as possible, we need some information.
+
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug description
+      description: What happened?
+    validations:
+      required: true
+
+  - type: input
+    id: os
+    attributes:
+      label: Operating system
+      description: "Which operating system are you on? Please provide the version as well. If you are on a Mac, please specify Apple silicon or Intel."
+      placeholder: "macOS Ventura 13.4 (Arm), Windows 11"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: install
+    attributes:
+      label: Installation Method
+      description: "How did you install AI-Lab ?"
+      options:
+        - "from `ghcr.io/containers/podman-desktop-extension-ai-lab` container image"
+        - "from Podman-Desktop extension page"
+        - "Other"
+
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of the software are you running?
+      options:
+        - "next (development version)"
+        - "1.0.0"
```

Comment:
in Podman Desktop the release job automate the update of this file, how it will be updated there ?

Reply:
✨Manually ✨